src/main/webapp/scheduled-interviews.js

```diff
@@ -12,6 +12,95 @@
+        const emptyEventsMessageDiv = document.createElement('div');
+        const  emptyEventsMessage = document.createElement('h2');
+        emptyEventsMessage.style.textAlign = 'center';  
+        emptyEventsMessage.innerHTML = "No Scheduled Interviews"; 
+        emptyEventsMessageDiv.appendChild(emptyEventsMessage); 
+        scheduledInterviewsSection.appendChild(emptyEventsMessage); 
+      } else { 
+        scheduledInterviews.forEach((scheduledInterview) => {
+          scheduledInterviewsSection.appendChild(createScheduledInterviewCard(scheduledInterview));
+          console.log(scheduledInterview); 
+        })
+      }
+    }); 
 }
+
+function createScheduledInterviewCard(scheduledInterview) {
+  // Check the role of the current user for the scheduledInterview
+  var email = "gswe@gmail.com";
+  var role; 
+  if(email === scheduledInterview.interviewerEmail) {
+    role = "Interviewer"; 
+  } else {
+    role = "Interviewee";
+  }
+  const scheduledInterviewElement = document.createElement('div');
+  scheduledInterviewElement.className = 'row';
+
+  const scheduledInterviewCard = document.createElement('div');
+  scheduledInterviewCard.className = 'card w-75 scheduled-interview-card';
+
+  const scheduledInterviewCardBody = document.createElement('div');
+  scheduledInterviewCardBody.className = 'card-body';
+
+  const scheduledInterviewCardTitle = document.createElement('h5');
+  scheduledInterviewCardTitle.className = 'card-title'; 
+  scheduledInterviewCardTitle.innerText = `Your role: ${role}`;
+
+  const scheduledInterviewCardText = document.createElement('p');
+  scheduledInterviewCardText.className = 'card-text'; 
+  scheduledInterviewCardText.innerText = createDateString(scheduledInterview); 
+
+  const scheduledInterviewCardAttendees = document.createElement('ul'); 
+  scheduledInterviewCardAttendees.className = 'list-group list-group-flush'; 
+
+  const scheduledInterviewCardInterviewee = document.createElement('li'); 
+  scheduledInterviewCardInterviewee.className = 'list-group-item'; 
+  scheduledInterviewCardInterviewee.innerText = `Interviewee Email: ${scheduledInterview.intervieweeEmail}`; 
+
+  const scheduledInterviewCardInterviewer = document.createElement('li'); 
+  scheduledInterviewCardInterviewer.className = 'list-group-item'; 
+  scheduledInterviewCardInterviewer.innerText = `Interviewer Email: ${scheduledInterview.interviewerEmail}`;
+
+  scheduledInterviewCardAttendees.appendChild(scheduledInterviewCardInterviewee); 
+  scheduledInterviewCardAttendees.appendChild(scheduledInterviewCardInterviewer); 
+  scheduledInterviewCardBody.appendChild(scheduledInterviewCardTitle); 
+  scheduledInterviewCardBody.appendChild(scheduledInterviewCardText); 
+  scheduledInterviewCard.appendChild(scheduledInterviewCardBody); 
+  scheduledInterviewCard.appendChild(scheduledInterviewCardAttendees);
+  scheduledInterviewElement.appendChild(scheduledInterviewCard); 
+  return scheduledInterviewElement; 
```

Comment:
Why are you returning here?

Reply:
This method returns a scheduledInterview html element (a div that contains the scheduled interview card)

Reply:
Gotcha. Wasn't paying close enough attention to where it is being called.